specs/MultiProfile.md

```diff
@@ -217,6 +232,76 @@ HRESULT AppWindow::OnCreateCoreWebView2ControllerCompleted(
 }
 ```
 
+### Manage password-autosave and general-autofill settings in profile
+
+```cpp
+HRESULT AppWindow::ManagePasswordAutosaveInProfile(ICoreWebView2Controller* controller)
+{
+    // ...
+
+    // Get the profile object.
+    wil::com_ptr<ICoreWebView2> coreWebView2;
+    CHECK_FAILURE(m_controller->get_CoreWebView2(&coreWebView2));
+    Microsoft::WRL::ComPtr<ICoreWebView2Profile> webView2Profile;
+    CHECK_FAILURE(webView2->get_Profile(&webView2Profile));
+    
+    // Get current value of password-autosave property.
+    BOOL enabled;
+    CHECK_FAILURE(webView2Profile->get_IsPasswordAutosaveEnabled(&enabled));
```

Comment:
This doesn't compile. `ICoreWebView2Profile` doesn't have a `get_IsPasswordAutosaveEnabled` method. Need to QI for `ICoreWebView2Profile3` first.

Reply:
Thanks for reminding, updated the code, thanks.